sources/packages/web/src/components/aest/students/modals/ApproveApplicationOfferingChangeRequestModal.vue

```diff
@@ -0,0 +1,109 @@
+<template>
+  <modal-dialog-base :showDialog="showDialog" title="StudentAid BC Declaration">
+    <template #content>
+      <p>
+        Please review and accept the StudentAid BC declaration to allow the
+        institution to make proposed changes to your application (on your
+        behalf).
+      </p>
+      <content-group-info class="my-3">
```

Comment:
As per Figma, I think it's a container with scrollable right? Maybe we can either user textbox or a container with scrollable if we are planning to follow the Figma
![image](https://github.com/bcgov/SIMS/assets/77353155/f6ce0013-eeb6-46a8-b2c2-4a11f0402cb4)